src/helm/common/human_task_request.py

```diff
@@ -0,0 +1,28 @@
+from dataclasses import dataclass
+from typing import Dict, List
+
+
+@dataclass(frozen=True)
+class HumanQuestionTemplate:
+    question_type: str  # multiple_choice, checkbox, or free_response
+    text: str
+    options: List[str]
+
+
+@dataclass(frozen=True)
+class HumanTaskTemplate:
+    name: str
+    instructions: str  # Can be HTML
+    num_workers: int
+    questions: List[HumanQuestionTemplate]
+
+
+@dataclass(frozen=True)
+class HumanTaskRequest:
+    template: HumanTaskTemplate
+    fields: Dict[str, str]
+
+
+@dataclass(frozen=True)
+class HumanTaskRequestResult:
+    answers: List[List[str]]
```

Comment:
Hard to reason about what this is - besides documentation, could this be a list of some class which contains a list?

Reply:
This is now a list of a dataclass.